### PR TITLE
Remove create in tisession

### PR DESCRIPTION
**Pull request by zhexuany**
In this PR, we propose remove `create()` in `TiSession` since we already have `getInstance()`

### CODE REVIEW

File: core/src/test/scala/com/pingcap/tispark/convert/ToUnsignedSuite.scala [hunk 1]
```diff
@@ -418,10 +422,9 @@ class ToUnsignedSuite extends BaseDataSourceTest("test_data_type_convert_to_unsi
...
 
-  override def afterAll(): Unit =
-    try {
-      dropTable()
-    } finally {
-      super.afterAll()
-    }
+  override def afterAll(): Unit = {
+    super.afterAll()
+    dropTable()
```
Comment (marsishandsome) [inline core/src/test/scala/com/pingcap/tispark/convert/ToUnsignedSuite.scala]: why change the order?

File: core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala [hunk 2]
```diff
@@ -37,6 +37,7 @@ class BaseDataSourceTest(val table: String,
   protected def tidbWrite(rows: List[Row],
                           schema: StructType,
                           param: Option[Map[String, String]] = None): Unit = {
+    refreshConnections()
```
Comment (marsishandsome) [inline core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala]: better to add `clear catalog cache` after each `drop table`

File: core/src/test/scala/com/pingcap/tispark/convert/ToUnsignedSuite.scala (hunk 1 above)
Comment (zhexuany) [reply core/src/test/scala/com/pingcap/tispark/convert/ToUnsignedSuite.scala]: It was already undo in my local.

File: core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala (hunk 2 above)
Comment (zhexuany) [reply core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala]: DONE

File: core/src/test/scala/org/apache/spark/sql/catalyst/plans/BasePlanTest.scala [hunk 3]
```diff
@@ -159,24 +159,4 @@ class BasePlanTest extends BaseTiSparkTest {
     assert(estimatedRowCount === answer)
   }
 
-  test("baseline test") {
```
Comment (birdstorm) [inline core/src/test/scala/org/apache/spark/sql/catalyst/plans/BasePlanTest.scala]: Why do we move this test?

Comment (zhexuany) [reply core/src/test/scala/org/apache/spark/sql/catalyst/plans/BasePlanTest.scala]: I think `test` should belong to a `suite` rather than `BasexxxTest`

**Pull request merged**